doc/api/http2.md

```diff
@@ -524,6 +524,33 @@ Used to set a callback function that is called when there is no activity on
 the `Http2Session` after `msecs` milliseconds. The given `callback` is
 registered as a listener on the `'timeout'` event.
 
+#### http2session.setConnectionWindowSize(windowSize)
+<!-- YAML
+added: v12.0.0
+-->
+
+* `windowSize` {number}
+* Returns: 0 
```

Comment:
Why return anything?

Reply:
In fact it's not exaclty the case, at this stage it returns only zero or throws ERR_OUT_OF_RANGE, the function is mapped to nghttp2_session_set_local_window_size. 
If the library changes its internal, the value will be returned.
If it does not fit the philosophy of node, we can return a generic errror that maps all errors at the native code level.